avalon/pipeline.py

```diff
@@ -246,6 +250,17 @@ def remove(self, container):
         raise NotImplementedError("Loader.remove() must be "
                                   "implemented by subclass")
 
+    @classmethod
+    def get_options(cls, contexts):
+        """
+            Returns static (cls) options or could collect from 'contexts'.
+
+            Args:
+                contexts (list): of repre or subset contexts
+            Returns:
+                (list)
+        """
+        return cls.options
```

Comment:
Maybe should check if `cls.options is None`. For cases when somebody will set the attribute value to `None` (to turn of options?) as `None` is used as checked for existence of options and would invalidate whole loading and skip the processing without telling what happened.